src/mobile/components/article/templates/sections/images.jade

```diff
@@ -0,0 +1,48 @@
+mixin printArtistName(artist)
```

Comment:
This seems to be pretty similar to the old `article/templates/sections/artworks.jade` file. I'm not 100% clear of the reason behind the name change.

Reply:
Yeah barely changed any code, but moved the artwork rendering into a mixin. The difference is that the old files cycled only through artworks, but we need to instead map an array containing a mix of artworks and images.